### PR TITLE
README revision

### PR DESCRIPTION
**Pull request by kelvinqian00**
Some revisions to make the README more organized and easier to maintain. (This doesn't affect the code or deployment so we won't be updating versions just yet.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,21 +10,35 @@ A SQUUID is a Universally Unique Identifier, or UUID, whose value increases stri
...
 
-If using deps.edn add the following line to your `:deps` map:
+If using deps.edn, add the following line to your `:deps` map:
 ```clojure
 com.yetanalytics/colossal-squuid {:mvn/version "0.1.3"}
 ```
+See the [Clojars page](https://clojars.org/com.yetanalytics/colossal-squuid) for how to install via Leiningen or other methods.
 
-If using Leiningen/Boot add the following dep:
+**Note:** By default colossal-squuid will bring in the Clojure and ClojureScript libraries as transitive dependencies. If you wish to exclude these from your project (e.g. because it is clj or cljs-only), you can use the `:exclusions` keyword (which works for both [deps.edn](https://simonrobson.net/2019/04/16/clojure-deps-with-exclusions.html) and [Leiningen](https://github.com/technomancy/leiningen/blob/master/sample.project.clj#L55)):
```
Comment (milt) [inline README.md:19]: This is common enough that I'm not sure it needs mentioning, but sure

Comment (kelvinqian00) [reply README.md:19]: Considering that the person who opened #13 moved the ClojureScript dependency to `:dev` I think it's reasonable to mention this.

File: README.md [hunk 2]
```diff
@@ -10,21 +10,35 @@ A SQUUID is a Universally Unique Identifier, or UUID, whose value increases stri
...
+(generate-squuid)
+;; => #uuid "017de28f-5801-8c62-9ce9-cef70883794a"
+
+(generate-squuid*)
+;; => {:timestamp #inst "2021-12-22T14:33:04.769000000-00:00"
+;;     :base-uuid #uuid "85335e1f-9c1f-4c62-9ce9-cef70883794a"
+;;     :squuid    #uuid "017de28f-5801-8c62-9ce9-cef70883794a"}
+
+(time->uuid #inst "2021-12-22T14:33:04.769000000-00:00")
+;; => #uuid "017de28f-5801-8fff-8fff-ffffffffffff"
```
Comment (milt) [inline README.md:41]: Usage examples make good docs _great_!

**Review by milt — APPROVED** [review]
These are great, points for taking the same basic information and conveying it more effectively!

**Pull request merged**